tests/disaster-recovery/regional-dr/test_failover_and_relocate.py

```diff
@@ -121,6 +131,39 @@ def test_failover_and_relocate(
         logger.info(f"Waiting for {wait_time} minutes to run IOs")
         sleep(wait_time * 60)
 
+        # Set cluster_kubeconfig value for 'drpc_obj' object to fetch the details without switching the cluster context
+        acm_cluster_kubeconfig = os.path.join(
+            config.clusters[get_active_acm_index()].ENV_DATA["cluster_path"],
+            config.clusters[get_active_acm_index()].RUN.get("kubeconfig_location"),
+        )
+        drpc_obj.cluster_kubeconfig = acm_cluster_kubeconfig
+
+        # Get lastGroupSyncTime before failover
+        drpc_data = drpc_obj.get()
+        last_group_sync_time = drpc_data.get("status").get("lastGroupSyncTime")
+        logger.info(
+            f"The value of lastGroupSyncTime before failover is {last_group_sync_time}."
+        )
+
+        # Verify lastGroupSyncTime before failover
+        time_format = "%Y-%m-%dT%H:%M:%SZ"
+        last_group_sync_time_formatted = datetime.strptime(
+            last_group_sync_time, time_format
+        )
+        current_time = datetime.strptime(
+            datetime.utcnow().strftime(time_format), time_format
+        )
+        time_since_last_sync = (
+            current_time - last_group_sync_time_formatted
+        ).total_seconds() / 60
+        logger.info(
+            f"Before failover - Time in minutes since the last sync {time_since_last_sync}"
+        )
+        assert (
+            time_since_last_sync < scheduling_interval + 2
```

Comment:
Instead of adding +2, it would be good to failover or even relocate when lastGroupSyncTime is within 2x range of sync interval. 
e.g. if last sync happened 10mins ago and sync interval is 5mins, we are good to failover/relocate. The difference shouldn't cross 2x.

Reply:
Done